gearcmd/worker.go

```diff
@@ -43,6 +44,11 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 		data["error_message"] = err.Error()
 		data["value"] = 0
 		data["success"] = false
+		if conf.RetryCount > 0 {
+			log.Println(kayvee.FormatLog("gearcmd", kayvee.Error, "RETRY", data))
```

Comment:
I don't think it'd be an issue per se, but it'd be kind of confusing to anyone looking at the logs. Is there a way to prevent that?